examples/earthquake-custom-symbol.js

```diff
@@ -33,7 +33,7 @@ const styleFunction = function (feature) {
   // the Placemark's name instead.
   const name = feature.get('name');
   const magnitude = parseFloat(name.substr(2));
-  const size = parseInt(10 + 40 * (magnitude - 5), 10);
+  const size = Math.floor(10 + 40 * (magnitude - 5));
```

Comment:
Why not
```js
const size = 10 + 40 * (Number(magnitude) - 5)
```
?

Reply:
Calling parseInt did the same thing as Math.floor does, maybe rounding is not required.

Reply:
The icons are offset from the center and don't respect the device pixel ratio, fixing this.
![image](https://github.com/user-attachments/assets/93f9c471-1473-4a75-a2d5-1f737b0f4386)